insights/parsers/netstat.py

```diff
@@ -681,12 +681,22 @@ def get_service(self, service):
         return [l for l in self.data if l.get("Process", None) and service in l["Process"]]
 
     def get_localport(self, port):
-        return [l for l in self.data if l.get("Local-Address-Port") and
-                port in l["Local-Address-Port"]]
+        list_conn = []
+        for line in self.data:
```

Comment:
Since line.get('Local-Address-Port') being fetched times, how about simplifying as:
```python
            local_port = line.get('Local-Address-Port')
            if local_port and ':*' not in local_port and\
                    int(port) == int(local_port.split(':')[-1]):
                list_conn.append(line)
```

Reply:
Done